config/ui/ui.go

```diff
@@ -22,12 +23,13 @@ var (
 
 // Config is the configuration for the UI of Gatus
 type Config struct {
-	Title       string   `yaml:"title,omitempty"`       // Title of the page
-	Description string   `yaml:"description,omitempty"` // Meta description of the page
-	Header      string   `yaml:"header,omitempty"`      // Header is the text at the top of the page
-	Logo        string   `yaml:"logo,omitempty"`        // Logo to display on the page
-	Link        string   `yaml:"link,omitempty"`        // Link to open when clicking on the logo
-	Buttons     []Button `yaml:"buttons,omitempty"`     // Buttons to display below the header
+	Title                  string   `yaml:"title,omitempty"`       // Title of the page
+	Description            string   `yaml:"description,omitempty"` // Meta description of the page
+	Header                 string   `yaml:"header,omitempty"`      // Header is the text at the top of the page
+	Logo                   string   `yaml:"logo,omitempty"`        // Logo to display on the page
+	Link                   string   `yaml:"link,omitempty"`        // Link to open when clicking on the logo
+	Buttons                []Button `yaml:"buttons,omitempty"`     // Buttons to display below the header
+	MaximumNumberOfResults int      // MaximumNumberOfResults to display on the page
```

Comment:
I'd rather not make this configurable at the UI level & let it be handled by the storage configuration. Since the UI supports pagination, we don't have to worry about this at the UI.

Reply:
Yes, I'm agree, that's why I did this https://github.com/TwiN/gatus/pull/476/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R262 It's a workaround to pass this value to ui, we need to to calculate number of pages, because it's hardcoded in the ui https://github.com/TwiN/gatus/pull/476/files#diff-1c806791cec1d6fff630b04440dfb90fff5a34ef3a17cadadfbeafa2c26dcfe2L4

Reply:
Ah that makes sense! Could you perhaps add a newline above `MaximumNumberOfResults` to separate it from the other parameters?

Also perhaps updating the documentation to make it clear why this field is different from the others (i.e. it's not configurable because we're passing it from the UI?)

---

_Alternatively, I wonder if we should just not pass this at all & when a user tries to go to a page with no results, just mentioned that there's no results._